### ISSUE TITLE
Failed to play this stream

### ISSUE THREAD

**Issue by tro69** [TeamNewPipe/NewPipe#883]
When I try to watch this [video](https://www.youtube.com/watch?v=4LfJnj66HVQ) I get this error message: "Failed to play this stream".
I also tried to play video with all video resolutions and to play in background (only audio) but I got same error message.
I'm running NewPipe 0.11.0 on Android 7.1.2

**Comment by theScrabi** [comment]
Confirmed.

**Comment by voidpublic** [comment]
can confirm as well.Seems to be happen randomly. Killing the app completely and restarting it seems to fix it temporarily

**Comment by tro69** [comment]
I tried many times to kill the app completely and restart it but it doesn't work anyway.

**Comment by SudoLinux** [comment]
@tro69 I think @voidpublic means that you need to kill via setting menu and not the app switcher... :)

**Comment by v0idst4r** [comment]
@SudoLinux I think he knew what he meant. This video still doesn't play when force closing (for me).

**Comment by v0idst4r** [comment]
I know other videos this happens to. I remember them working on old versions of newpipe though.

**Comment by v0idst4r** [comment]
Could it be DRM related?

**Comment by voidpublic** [comment]
Yeah can confirm, been seeing it now as well. 

This video for example: https://www.youtube.com/watch?v=lvs68OKOquM

It does work when killing them app, sometimes for a long time (have been going for 3 weeks without that issue). Recently (like last 2 days) I've had it happen multiple times a day. Exact same video. Happens to background/fullplay and popout. 


If you can let me know how to enable debugging i'm happy to provide logs.

**Comment by sensiblepuffin** [comment]
This happened to me last night, so I tried logging with VLC to try and figure out what was going on. (Settings -> Advanced -> Debug logs)

Log: https://pastebin.com/t6N6T4DP

It looks like it's getting a 403 Forbidden in response to its GET. However, I was using a VPN when the error occurred, so this might have been the endpoint getting blocked for making requests too often. By the time I realized this, however, I stopped getting the error.

Can someone else try to replicate what I see while not using a VPN?
